nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -475,6 +475,8 @@ public Description matchAssignment(AssignmentTree tree, VisitorState state) {
           state,
           ASTHelpers.getSymbol(tree.getVariable()));
     }
+    handler.handleNonnullFieldAssignment(
```

Comment:
Capitalizing null looks better to me:
```suggestion
    handler.handleNonNullFieldAssignment(
```

Reply:
Sure, should we also maybe rename it to `onNonNullFieldAssignment` to match other existing APIs in the handler ?

Reply:
Sure `onNonNullFieldAssignment` is even better

Reply:
9e38931559f711f3c130c152d6dfbca70c70eaa3